CHANGELOG.md

```diff
@@ -27,12 +27,16 @@ Changelog
 -   Support non-gate instructions (e.g. `MEASURE`) in `to_latex()` (@notmgsk, gh-975).
 -   Test suite has been updated to reduce the use of deprecated features
     (@kilimanjaro, gh-998).
+-   Type hints have been added to `pyquil.gates` and `pyquil.quilatom` modules (@appleby gh-999).
```

Comment:
comma

Reply:
Also 999 -- I wonder who is gonna get 1000.

Reply:
It's like watching the odometer roll over in your car